app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,61 @@
+class AlbumsController < ApplicationController
+  def index
+    @model = Album.all
+    @path = new_album_path
+    @add_button = "Add New Album"
+  end
+
+  def show
+    @media = Album.find(params[:id])
```

Comment:
Not sure if you have covered this in class already, but this would be a great place to use a before_action to DRY up your code. Lines 9, 35, and 46 are all doing the same thing (`Album.find(params[:id])`). If you create a before_action to set an instance variable equal to `Album.find(params[:id])`, then you can eliminate these lines. Here's a link to the docs:

http://guides.rubyonrails.org/action_controller_overview.html#filters